src/presentation/services/event-builder.ts

```diff
@@ -128,7 +129,8 @@ export class EventBuilder implements RundownEventBuilder, ActionTriggerEventBuil
       type: IngestEventType.RUNDOWN_CREATED,
       timestamp: Date.now(),
       rundownId: rundown.id,
-      basicRundown: new BasicRundownDto(rundown)
+      basicRundown: new BasicRundownDto(rundown),
+      rundown: new RundownDto(rundown)
```

Comment:
Do you still need the `basicRundown`? All the values that the BasicRundown have is also in the Rundown

Reply:
Nope, i removed it, and added the changes to the frontend as well!